lib/api/CDataFrameAnalysisInstrumentation.cc

```diff
@@ -153,6 +162,18 @@ void CDataFrameAnalysisInstrumentation::monitorProgress(
             lastProgress = progress;
             writeProgress(lastTask, lastProgress, &writer);
         }
+        LOG_DEBUG(<< "Memory usage " << instrumentation.memory());
+        if (instrumentation.memory() > instrumentation.m_MemoryLimit) {
+            LOG_DEBUG(<< "Memory limit triggered");
```

Comment:
Don't think this is needed with the other
```suggestion
```

Reply:
Deleted